documentation-site/components/version-selector.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+/* eslint-disable flowtype/require-valid-file-annotation */
+/* eslint-env browser */
+
+import * as React from 'react';
+import semver from 'semver';
+
+import ChevronDown from 'baseui/icon/chevron-down';
+import {StatefulPopover, PLACEMENT as PopoverPlacement} from 'baseui/popover';
+import {StatefulMenu, NestedMenus} from 'baseui/menu';
+import {Button, KIND} from 'baseui/button';
+
+import {version} from '../../package.json';
+import versions from '../../versions.json';
+
+const MAJOR_VERSIONS = [
+  {label: 'v8'},
+  {label: 'v7'},
+  {label: 'v6'},
+  {label: 'v5'},
+  {label: 'v4'},
+  {label: 'v3'},
+  {label: 'v2'},
+  {label: 'v1'},
+];
+
+const VersionSelector = () => {
+  const versionsToShow = versions
+    .filter(releaseVersion => {
+      // we have "now" deployments since v8.1.0
+      return semver.satisfies(releaseVersion.name, '>=8.1.0');
+    })
+    .map(item => {
+      return {
+        label: item.name,
+        commit: item.target_commitish,
+      };
+    });
+
+  return (
+    <StatefulPopover
+      placement={PopoverPlacement.bottomLeft}
+      dismissOnClickOutside={false}
+      content={({close}) => (
+        <NestedMenus>
+          <StatefulMenu
+            items={MAJOR_VERSIONS}
+            onItemSelect={({item}) => {
+              window.open(`https://${item.label}.baseweb.design`);
+              close();
+            }}
+            overrides={{
+              List: {
+                style: {
+                  width: '84px',
+                },
+              },
+              Option: {
+                props: {
+                  size: 'compact',
+                  getChildMenu: item => {
+                    if (
+                      semver.satisfies(semver.coerce(item.label), '>=8.0.0')
+                    ) {
+                      return (
+                        <StatefulMenu
+                          size="compact"
+                          items={versionsToShow}
+                          onItemSelect={({item}) => {
```

Comment:
I thought this was not working?

Reply:
it doesn't work if I remove `dismissOnClickOutside={false}`